argocd/run.sh

```diff
@@ -54,9 +54,8 @@ ARCH=$(uname -m)
 if [ "$ARCH" == "x86_64" ]; then
     SYSTEM_ARCH="amd64"
 else
-    if [ "$ARCH" == "aarch64" ]; then
-        SYSTEM_ARCH="arm64"
-    fi
+    # Default to arm64 because community bundle don't support arm container so uname will always returns x86_64 
+    SYSTEM_ARCH="arm64"
```

Comment:
if `uname` always returns `x86_64`, this block is never used, no? (see the `if` above)

Reply:
Good catch, removed